### ISSUE TITLE
[question] 4.X release and question about release cycles

### ISSUE THREAD

**Issue by mhk-etn** [dailymotion/vast-client-js#410]
Hi,

I skimmed open/closed threads and haven't seen this question in particular posted before explicitly so I figured I would ask just for clarification.

I assume there isn't any solid plans yet for version 4.X release or some form of RC since work is mostly being done on the 3.X branch now and from what I gathered from other issues, changes and p. requests which end up in the 4.X branch are mostly due to them being breaking changes to the currently 3.X version being worked on so it's safe to assume that 4.X branch is not going to be released anytime soon and should be build for usage individually?

TL;DR branch 4.X is for breaking changes and is not going to be released anytime soon...

MHk

**Comment by ZacharieTFR** [comment]
Hello @mhk-etn, 
Yes, you are right, we haven't defined a RC yet, so to benefit from the breaking change you need to build/use the 4.X but we will release the 4.X soon.
